commit b7a4c9488cc2b00ee019ea03016a65027d4e9018
Author: Dmitry Smirnov <divis1969@gmail.com>
Date:   Fri Apr 15 22:51:05 2016 +0300

    Add MTK-specific APIs of the audio HAL
    
    These APIs allow Engineering app to read/write audio data from/to NV
    
    Change-Id: Ic1b59119081b41b8d1f59abfbf24a33b3d97a7a6

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index c3cb740..02372f5 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -753,6 +753,12 @@ struct audio_hw_device {
     int (*listen_set_parameters)(struct audio_hw_device *dev,
                                  const char *kv_pairs);
 #endif
+    int (*dummy1)(void);
+    int (*dummy2)(void);
+    int (*dummy3)(void);
+    int (*dummy4)(void);
+    int (*set_audio_data)(struct audio_hw_device *dev, int par1, size_t len, void *ptr);
+    int (*get_audio_data)(struct audio_hw_device *dev, int par1, size_t len, void *ptr);
 };
 typedef struct audio_hw_device audio_hw_device_t;
 
